packages/jellyfish-api-core/__tests__/category/mining/getMiningInfo.test.ts

```diff
@@ -28,9 +28,9 @@ describe('Mining', () => {
     expect(info.difficulty).toBeDefined()
     expect(info.isoperator).toStrictEqual(true)
 
-    expect(mn1.masternodeid).toBeDefined()
-    expect(mn1.masternodeoperator).toBeDefined()
-    expect(mn1.masternodestate).toStrictEqual('ENABLED')
+    expect(mn1.id).toBeDefined()
+    expect(mn1.operator).toBeDefined()
+    expect(mn1.state).toStrictEqual('ENABLED')
```

Comment:
this is old structure, is something wrong with docker image??

Reply:
initially, the branch target for `1.8.x` however I cherry-picked and these changes were dropped as 1.8.x wasn't ready yet.